doc/classes/RenderingServer.xml

```diff
@@ -3314,7 +3314,7 @@
 			<return type="void" />
 			<param index="0" name="generate" type="bool" />
 			<description>
-				This method is currently unimplemented and does nothing if called with [param generate] set to [code]true[/code].
+				This method is used to ensure that the Compatibility renderer generates debug wireframes for all meshes that are loaded. By default, the engine won't generate debug wireframes at run time since they slow down loading of assets and take up VRAM. You must enable debug wireframes with this method if you wish to use wireframes in the Compatibility renderer.
```

Comment:
If I understand this right:
```suggestion
				If [param generate] is [code]true[/code], generates debug wireframes for all meshes that are loaded. By default, the engine does not generate debug wireframes at runtime, since they slow down loading of assets and take up VRAM.
[b]Note:[/b] This method is also needed to enable wireframes in the Compatibility renderer. 
```

Reply:
We could also add a note in the `Viewport.DEBUG_DRAW_WIREFRAME` and `RenderingServer.VIEWPORT_DEBUG_DRAW_WIREFRAME` descriptions.

Reply:
Updated docs, including for `VIEWPORT_DEBUG_DRAW_WIREFRAME` and `DEBUG_DRAW_WIREFRAME`